CHANGELOG.md

```diff
@@ -77,7 +77,9 @@ END_UNRELEASED_TEMPLATE
 
 {#v0-0-0-added}
 ### Added
-* Nothing added.
+* Repo utilities `execute_unchecked`, `execute_checked`, and `execute_checked_stdout` now
```

Comment:
The repo_utils apis are internal, so no need to document this in the user-facing changelog.

Reply:
Ack :+1:, I'll keep that in mind for next time.